tests/e2e/resolve-repositories.test.js

```diff
@@ -0,0 +1,152 @@
+import { resolveRepositories } from '../../lib/resolve-repositories.js';
```

Comment:
it's not an end-to-end test if you test an internal library. I think we can leave out the e2e tests entirely for now. The tests are actually sending requests to the real API which is not something we can do in tests, it will fail randomly, especially in CI, especially when sending unauthenticated requests